dev-libs/openssl/metadata.xml

```diff
@@ -8,7 +8,6 @@
 <use>
  <flag name="asm">Support assembly hand optimized crypto functions (i.e. faster run time)</flag>
  <flag name="bindist">Disable/Restrict EC algorithms (as they seem to be patented) -- note: changes the ABI</flag>
- <flag name="ktls">Enable support for Kernel implementation of TLS (kTLS)</flag>
```

Comment:
I guess you could keep this line. I know it's out of date, but that would minimize the diff with upstream.